packages/grafana-data/src/types/options.ts

```diff
@@ -39,7 +39,7 @@ export interface OptionEditorConfig<TOptions, TSettings = any, TValue = any> {
   /**
    * Array of strings representing category of the option. First element in the array will make option render as collapsible section.
    */
-  category?: string[];
+  category?: Array<string | undefined>;
```

Comment:
Out of curiosity, when would you have an undefined (sub)category?

Reply:
In a case when you want to have subcategories under the main category of the field options, see **Field options sub-categories in the main field options pane** in the PR description.